distribution/lib/Standard/Database/0.0.0-dev/src/Data/Dialect.enso

```diff
@@ -52,10 +52,24 @@ type Dialect
 
    A vector of SQL dialects supported by the Database library.
 supported_dialects : Vector Dialect
-supported_dialects = [Postgres.postgresql, Sqlite_Module.sqlite, Redshift.redshift]
+supported_dialects = [Postgres_Module.postgresql, Sqlite_Module.sqlite, Redshift_Module.redshift]
```

Comment:
With the new approach to creating connections, is this list still used? Unless I missed something, it feels obsolete.